docs/central-upgrade.rst

```diff
@@ -3,52 +3,90 @@
 Upgrading Central
 =================
 
-We release new versions of Central regularly. You do not have to upgrade to the latest version immediately, but we generally recommend that you do so to get access to the newest features, bug fixes, and security updates.
+We release new versions of Central regularly. We generally recommend that you upgrade as soon as you can to get access to the newest features, bug fixes, and security updates. It is not generally possible to downgrade and we do not guarantee that versions prior to the latest release can still be installed.
 
-.. admonition:: Note
+Start by reviewing upgrade notes for all versions between your current version and the one you are upgrading to.
 
-  To see your version number, click on the question mark icon in the upper right section of your Central menu bar, then click :guilabel:`Version`. If you don't see the question mark, you can also see the version number by adding ``version.txt`` to the root URL (e.g., `demo.getodk.cloud/version.txt <https://demo.getodk.cloud/version.txt>`_).
+* Central v1.5: no special instructions.
```

Comment:
Why mention v1.5 in particular?

Reply:
People on the forum sometimes ask if there are upgrade instructions for a version if we don't explicitly document that there are none.

Reply:
That makes sense to me. I guess I'm wondering why we'd mention v1.5 and not other versions, for example, v2022.3 or v1.1. Could we instead add a generic note along the lines of "Most versions of Central do not have special upgrade instructions"?

Reply:
(Also, definitely don't want to block merging / no strong feelings, mostly just wanted to throw out the thought.)